definitions/npm/react-modal_v3.1.x/flow_v0.54.1-/react-modal_v3.1.x.js

```diff
@@ -33,7 +33,7 @@ declare module 'react-modal' {
     },
     appElement?: HTMLElement | string | null,
     onAfterOpen?: () => void | Promise<void>,
-    onRequestClose?: () => void,
+    onRequestClose?: (SyntheticMouseEvent<> | SyntheticKeyboardEvent<>) => void,
```

Comment:
Does it make sense to generalize this to `SyntheticEvent<>`?

Reply:
I thought it may as well define concrete type when this handler has clear type to handle...😛
But it's easier to use generic event type for handling events.

Thank you for your advice, I modified it.